index.html

```diff
@@ -0,0 +1,114 @@
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <link href="https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
+    integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <meta charset="utf-8">
+    <title>Backgrounds and Gradients</title>
+  </head>
+
+  <body>
+
+    <header>
+      <!-- navegator bar here-->
+      <section>
+        <nav id="globalHeader">
+          <ul id="globalNav">
+              <li id="icon-apple">
+                <span><i class="fab fa-apple"></i></span>
+              </li><!-- mac icon-->
+              <li id="icon-store">Store</li>
+              <li id="icon-mac">Mac</li>
+              <li id="icon-iPod">iPod</li>
+              <li id="icon-iPhone">iPhone</li>
+              <li id="icon-iPad">iPad</li>
+              <li id="icon-iTunes">iTunes</li>
+              <li id="icon-support">Support</li>
+          </ul>
+            <div class="search-Nav">
+              <form class="searchForm" action="index.html" method="get">
+                <div class="searchIcon"><i class="fas fa-search"></i></div>
+                <input type="text" id="searchInput" size="6">
+              </form>
+            </div>
+        </nav>
+      </section>
+    </header>
+
+    <main>
+          <div class="background">
+        <section class="mainWrapper">
+          <h1 class="mainTitle">What will your verse be?</h1>
+          <h2 class="mainDescription row1">Everyone has something to share. See how people are using</h2>
+          <h2 class="mainDescription row2">iPad in their unique ways to add to the world’s story.</h2>
+          <div class="main link">
+          <a href="#">
+            <p id="linkText">Watch the “Your Verse” film and explore the stories <i class="far fa-play-circle"></i></p>
+          </a></div>
+        </section>
+      </div>
+    </main>
+
+      <section class="globalPromos">
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140419112642im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg" alt="iPhone5s">
+        </figure>
+
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140419112642im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone5c">
+        </figure>
+
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140226015601im_/http://images.apple.com/home/images/promo_30_years_2x.jpg" alt="promoBirthday">
+        </figure>
+
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140223180930im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="promoSupplier">
+        </figure>
+
+      </section>
```

Comment:
Move this section tag into the footer and edit it with the same guideline given for the previous section tag.

Reply:
Hi! how are you? thank you for your code review :)  I'm not agree that this belongs to the footer :) I think the footer is only the part of the "Shop the Apple Online Store..." Thanks.